index.html

```diff
@@ -21,6 +20,6 @@ <h1>Awesome Books</h1>
         <button class="add-btn" type="submit">Add</button>
       </form>
```

Comment:
### Your Design
![student-design](https://user-images.githubusercontent.com/93039359/182342786-73500dbe-2d81-4e3e-9076-696fa1cc79e9.png)
### Basic Wireframe
<img width="437" alt="basic-wireframe" src="https://user-images.githubusercontent.com/93039359/182342239-a69a2886-6380-47bc-8b41-1ed53ca43acf.png">


- [ ] You have made great progress on this project but you are not following the wireframe. Please make your design according to the following wireframe :+1: